pkg/metrics/metrics.go

```diff
@@ -0,0 +1,95 @@
+package metrics
+
+import (
+	"github.com/aquasecurity/tracee/tracee-ebpf/tracee"
+	"github.com/aquasecurity/tracee/tracee-rules/engine"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+func RegisterTraceeMetrics(t *tracee.Tracee) error {
+	err := prometheus.Register(prometheus.NewCounterFunc(prometheus.CounterOpts{
```

Comment:
why not using sugar "notation" `if err := prometheus.Register(...); err!=nil{}`

Reply:
conversation exists below.